### PR TITLE
Fix amputee bug in bodymod; restore missing bovine tats to bodyMod, fix many missing bodymod checks in longSlaveDesc

### PR DESCRIPTION
**Pull request by lowercase-donkey**
From [reddit](https://www.reddit.com/r/freecitiesgame/comments/6oc7sk/not_sure_if_bug_or_purposeful_omission/).

Edit: while in here, I found I failed to add bovine tats when I overhauled bodymod, I think.  Whoops.

### CODE REVIEW

**Comment by lowercase-donkey** [comment]
Brings up the question though, why are vagina tats and abdomen tats confused?  The vaginaTat really is just abdomen tats, albeit ones that sometimes refer to the vagina.

We even have this segment:
`$possessiveCap abdomen is tattooed with tribal patterns that draw the eye down toward $possessive <<if $activeSlave.dick != 0>>cock<<elseif $activeSlave.vagina == -1>>featureless crotch<<else>>vagina<</if>>.`

How the heck is that a 'vagina' tattoo.

To avoid breaking a lot of slaves, would it be fine to leave the variable name in code, add a coded labiaTat, and change the displayed slot from Vagina to Abdomen?

**Comment by lowercase-donkey** [comment]
My fault.  Not sure how I missed having it.

That exposed another problem in the way things are handled in bovine, though, which the new if conditional checks.  Basically, bovine's shoulder slot describes arms and upper/lower back, and then the ACTUAL arms and back slots are empty.  This produces white lines in the way bodyMod was calling widget descriptions.

This is a kludge on a kludge, though.  It might be better to simply (boringly) describe her cow tats in the same way as the others, slot by slot.

**Comment by lowercase-donkey** [comment]
This commit reflects the old idea that bovine is an all or nothing tat, which gives a rationale for shoulders.  

It also avoids confused players giving cow tats to arms alone, and never seeing that described (because they need to tattoo shoulders instead.)

Do we need to stick to this though?

**Comment by lowercase-donkey** [comment]
Oh shit, this still won't work.  We still have the problem where someone can apply bovine, then try to do a back or arm tattoo.  Not only will it apply but not show, but the player will see shoulderTat as "overriding" what they chose. 

[When I have time, I'll add some boring cow descriptions.](http://www.portal2sounds.com/620)

**Comment by lowercase-donkey** [comment]
Fixed a lot of needed <<if $showBodyMods == 1>>.  It was so leaky, I didn't realize I had it set to 0 in my game, confusing the heck out my work for a bit.

File: src/uncategorized/descriptionWidgetsTattoos.tw [hunk 1]
```diff
@@ -190,75 +190,79 @@
...
+	<<elseif $activeSlave.backTat == "counting">>
+		Her back is tattooed with tick marks to count the days of her sexual slavery
+		<<set $seed = (($week-$activeSlave.weekAcquired)*7)+random(-3,3)>>
+		<<if $seed < 10>>
+		  here. There are only a few.
+		<<else>>
+		  here: $seed of them.
+		<</if>>
+	<<elseif $activeSlave.shouldersTat == "bovine patterns">>
+		$possessiveCap back is covered in a large saddle-like blotch.
```
Comment (freecitiesbandit) [inline src/uncategorized/descriptionWidgetsTattoos.tw]: <<elseif $activeSlave.shouldersTat == "bovine patterns">>
		$possessiveCap back is tattooed to resemble the dappled flank of a cow.

File: src/uncategorized/descriptionWidgetsTattoos.tw [hunk 2]
```diff
@@ -301,6 +305,8 @@
 			$pronounCap has a tramp stamp which reads 'Fuck my ass and be blessed.'
 		<<case "paternalist">>
 			$pronounCap has a tramp stamp which reads 'Fuck me slowly' with an arrow pointing downward.
+		<<case "bovine patterns">>
+			Spots run right down to where her tail would be.
```
Comment (freecitiesbandit) [inline src/uncategorized/descriptionWidgetsTattoos.tw]: <<case "bovine patterns">>
 			$pronounCap has an identifying barcode tattooed across $possessive lower back.

File: src/uncategorized/descriptionWidgetsTattoos.tw [hunk 3]
```diff
@@ -549,5 +555,13 @@
 		$possessiveCap legs are covered in tattoos of swirling flames.
 	<<case "paternalist">>
 		$possessiveCap left thigh reads 'I Love My Life,' and the right 'It's All I'm Good For.'
+	<<case "bovine patterns">>
+		$possessiveCap legs are completely colored to look like a cow, from her hips to the tops of her feet.
```
Comment (freecitiesbandit) [inline src/uncategorized/descriptionWidgetsTattoos.tw]: <<case "bovine patterns">>
 		$possessiveCap legs are covered in tattoos of spots, giving $possessive a noticeably bovine appearance.

File: src/uncategorized/descriptionWidgetsTattoos.tw [hunk 4]
```diff
@@ -190,75 +190,79 @@
 		<<else>>
 		  Tiny dick, pussy, and anus symbols are tattooed all over both her arms, one for each organ she's manually pleasured.
 		<</if>>
+	<<elseif $activeSlave.armsTat == "bovine patterns">>
+		Cow-like spots go down her arms, but stop short of each of her elbows.
```
Comment (freecitiesbandit) [inline src/uncategorized/descriptionWidgetsTattoos.tw]: <<elseif $activeSlave.armsTat == "bovine patterns">>
		Tattoos of cow-like spots cover her arms, but stop short of each of her elbows.

File: src/uncategorized/descriptionWidgetsTattoos.tw [hunk 5]
```diff
@@ -23,8 +23,8 @@
 	'Hurt Me' is tattooed on her left shoulder, and 'Rape Me' on her right.
   <<elseif $activeSlave.shouldersTat == "counting">>
 	¤ tattoo counting her earnings and acquirements cover her shoulders.
-  <<elseif ($activeSlave.armsTat == "bovine patterns") && ($activeSlave.legsTat == "bovine patterns") && ($activeSlave.shouldersTat == "bovine patterns") && ($activeSlave.backTat == "bovine patterns")>>
-	Bovine tattoos cover her back and shoulders, and go down her arms, but stop short of each of her elbows.
+  <<elseif $activeSlave.shouldersTat == "bovine patterns">>
+	Bovine tattoos cover her shoulders.
```
Comment (freecitiesbandit) [inline src/uncategorized/descriptionWidgetsTattoos.tw]: Her shoulders are flecked with tattoos of bovine spots and blotches.

**Comment by lowercase-donkey** [comment]
@freecitiesbandit Excellent improvements, thanks!  Do I need to do anything, or will they just apply when this is merged?

**Comment by freecitiesbandit** [comment]
@lowercase-donkey  No worries. I just submitted a review, so you would have to had then went on to edit your commits to include the suggested changes. Since it's already merged, I'll just start a new pull request with the changes.

**Comment by lowercase-donkey** [comment]
Interesting.  No way to just approve the review from git?

**Pull request merged**